esmvaltool/cmorizers/obs/cmorize_obs_cds_satellite_soil_moisture.ncl

```diff
@@ -61,73 +73,99 @@ begin
   ; Type
   TYPE = "sat"
 
-  ; Version
-  VERSION = "CUSTOM-TCDR-ICDR-20200602"
-
   ; Global attributes
   SOURCE = "https://cds.climate.copernicus.eu/cdsapp#!/" + \
     "dataset/satellite-soil-moisture"
   REF = "Gruber et al., Earth Syst. Sci. Data, " + \
     "doi: 10.5194/essd-11-717-2019, 2019."
   COMMENT = ""
 
+  SENSORS_FILEBASENAMES = (/"C3S-SOILMOISTURE-L3S-SSMV-COMBINED", \
+                            "C3S-SOILMOISTURE-L3S-SSMV-PASSIVE", \
+                            "C3S-SOILMOISTURE-L3S-SSMS-ACTIVE"/)
+  ; Version
+  VERSIONS = (/"COMBINED", "PASSIVE", "ACTIVE"/)
+  PATTERNS = (/"????000000-?CDR-v201?12.0.0.nc", \
+               "????000000-?CDR-v201?12.0.0.nc", ".nc"/)
+
```

Comment:
The pattern for active sensor version should be identical to the other two. At the moment it does not work for the active version